tests/integration/components/supplementary-file-browser-test.js

```diff
@@ -67,16 +112,16 @@ test('has additional files', function(assert) {
     assert.equal(this.$('#rightArrow').length, 1);
     assert.equal(this.$('#downArrow').length, 1);
 
+    // TODO enable when when tests are fixed
```

Comment:
Quick question and then I will return code reviewing: Can you (@binoculars) provide some clarity as to why a lot of tests here are being skipped? What do you mean by when tests are fixed and why is that not within the scope of this pr?